wiki/content/mutations/index.md

```diff
@@ -1021,6 +1021,84 @@ Result:
 }
 ```
 
+### JSON Upsert Example
+
+Let's consider those previous [examples](#example) but, instead of `rdf`, we are going to use `json`.
+
+The first example is to update the `name` information of an existing user, and if the user doesn't exist, we create a user and update `email` and `name` information.
```

Comment:
Another heading is probably not necessary, you could just add this example next to the RDF example.